static/js/settings.js

```diff
@@ -164,6 +164,22 @@ function _setup_page() {
         });
     }
 
+    var old_statuses = {};
+    if (keep_status) {
```

Comment:
They don't invoke `_setup_page` though, only `setup_page`. Still could be mistake here.